tests/metrics/functional/test_regression.py

```diff
@@ -1,89 +1,17 @@
-from functools import partial
-from math import sqrt
-
 import numpy as np
 import pytest
 import torch
 from skimage.metrics import (
     peak_signal_noise_ratio as ski_psnr,
     structural_similarity as ski_ssim
 )
-from sklearn.metrics import (
-    mean_absolute_error as mae_sk,
-    mean_squared_error as mse_sk,
-    mean_squared_log_error as msle_sk
-)
 
 from pytorch_lightning.metrics.functional import (
-    mae,
```

Comment:
Why don't we test them any more? I think we should still test the functionals as a separate composition to be correct.

Reply:
You are probably right. I try adding them back in a more meaningful way.

Reply:
I have added back tests just for functional, but this time right beside the corresponding class based tests such that the same parameterization could be reused for functional test